components/kyma-environment-broker/internal/storage/driver/postsql/operation.go

```diff
@@ -441,11 +441,9 @@ func calFailedStatusForOrchestration(entries []dbmodel.OperationStatEntry) ([]st
 		resultPerInstanceID[entry.InstanceID] = append(resultPerInstanceID[entry.InstanceID], entry.State)
 	}
 
-	log.Info("calFailedStatusForOrchestration() show resultPerInstanceID", resultPerInstanceID)
 	var invalidFailed, failedFound bool
 
 	for instanceID, statuses := range resultPerInstanceID {
-		log.Info("calFailedStatusForOrchestration() loop resultPerInstanceID", instanceID, statuses)
```

Comment:
What is the content of statuses and resultPerInstanceID? Could you share a snippet with example?

Reply:
My mistake!